port: [1, 2, 3, 4, 5, 6]
starboard: [7, 8, 9, 10, 11, 12]
front_depth: [13, 14, 15, 16, 17, 18]
back_depth: [19, 20, 21, 22, 23, 24]
front_heading: [25, 26, 27, 28, 29, 30]
back_heading: [31, 32, 33, 34, 35, 36]